book/_toc.yml

```diff
@@ -42,6 +42,7 @@
             - file: tutorials/database/3_get_spiral_example
             - file: tutorials/database/4_plot_raster_example
         - file: tutorials/microstructure/microstructure-tutorial
+        - file: tutorials/gpr/gpr
```

Comment:
since you have two notebooks now, you'll want to add a 'sections' block to point to each notebook: 
```
        - file: tutorials/gpr/index
          sections:
            - file: tutorials/gpr/gpr
            - file: tutorials/gpr/get_gpr_and_density
```

Reply:
I will just delete get_gpr_and_density.ipynb This was the template code provided by Micha, which I cobbled a bit.